doc/source/train/dl_guide.rst

```diff
@@ -623,6 +567,63 @@ directory <train-log-dir>` of each run.
     print(result.checkpoint.get_internal_representation())
     # ('local_path', '/home/ubuntu/ray_results/TorchTrainer_2022-06-24_21-34-49/TorchTrainer_7988b_00000_0_2022-06-24_21-34-49/checkpoint_000003')
 
+.. _train-result-object:
+
+Accessing Training Results
```

Comment:
Is this supposed to be nested inside 'logging'?

Also I think we should move Data Ingest below this section